packaging/deb.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "k6",
+    "maintainer": "Load Impact AB",
+    "description": "Load testing for the 21st century",
+    "changelog-cmd": "cat './release notes/v0.21.0.md'",
```

Comment:
Do we need this? We'd have to change it for every release and I'm not sure how pretty the markdown file intended for github would be when you use it as a deb changelog. Also, if I remember correctly, debian changelogs included the full history of the package, not just the changes in the latest release.

Reply:
Just saw the command in the `rpm.json` spec, that one looks much more maintainable

Reply:
missed the `version` variable, I'll added it